Dockerfile

```
@@ -1,5 +1,5 @@
 # Builder image
-FROM golang:1.15 as builder
+FROM golang:1.16.0 as builder
```

```suggestion
FROM golang:1.16 as builder
```
While less repeatable, sticking to the minor (not patch) release will minimize maintenance PR burden like this one.